python/ray/worker.py

```diff
@@ -18,6 +18,7 @@
 import threading
 import time
 import traceback
+import xarray as xr
```

Comment:
Like Philipp said, we don't want Ray to depend on xarray, so let's remove this import and put it somewhere else.